plugins/chat/spec/system/uploads_spec.rb

```diff
@@ -39,6 +39,35 @@
       expect(Chat::Message.last.uploads.count).to eq(1)
     end
 
+    it "adds a thumbnail for large images" do
+      SiteSetting.create_thumbnails = true
+
+      chat.visit_channel(channel_1)
+      file_path = file_from_fixtures("huge.jpg", "images").path
+
+      attach_file(file_path) do
+        channel_page.open_action_menu
+        channel_page.click_action_button("chat-upload-btn")
+      end
+
+      channel_page.send_message("thumbnail test")
+
+      expect(page).not_to have_css(".chat-composer-upload")
```

Comment:
```suggestion
      expect(page).not have_no_css(".chat-composer-upload")
```

I recommend a read to our internal Systematic System Specs dev topic to know the reason